docs/articles/documentation/using-testcafe/programming-interface/runner.md

```diff
@@ -468,7 +468,14 @@ You can also cancel all pending tasks at once using the [runner.stop](#stop) fun
 The quarantine mode is designed to isolate *non-deterministic* tests (that is, tests that sometimes pass and fail without any apparent reason)
 from the rest of the test base (*healthy* tests).
 
-In this mode, a failed test is executed several times. The test result depends on the outcome (passed or failed) that occurs most often. That is, if the test fails on most attempts, the result is failed. If the test result differs between test runs, the test is marked as unstable.
+When the quarantine mode is enabled, tests run according to the following logic:
+
+1. A test runs at the first time. If it passes, TestCafe proceeds to the next test. If the test fails, it enters the quarantine mode.
```

Comment:
> If the test fails, it enters the quarantine mode.

No, test run is already in quarantine mode.
It's better  to combine two this sentence and next as follows:
`If the test fails then test runs over and over....`